src/engagement_db_to_analysis/traced_data_filters.py

```diff
@@ -42,14 +43,17 @@ def rqa_time_range_filter(user, messages_traced_data, pipeline_config):
     filtered = []
     for td in messages_traced_data:
         if td["dataset"] in rqa_engagement_db_datasets:
-            if start_time_inclusive <= isoparse(td["timestamp"]) < end_time_inclusive:
-                td.append_data(td, Metadata(user, Metadata.get_call_location(), time.time()))
-                filtered.append(td)
+            if start_time_inclusive is not None and isoparse(td["timestamp"]) < start_time_inclusive:
+                continue
+            if end_time_inclusive is not None and isoparse(td["timestamp"]) > end_time_inclusive:
+                continue
+            td.append_data(td, Metadata(user, Metadata.get_call_location(), TimeUtils.utc_now_as_iso_string()))
+            filtered.append(td)
         else:
             filtered.append(td)
 
     log.info(f"Filtered out messages sent outside the time range "
```

Comment:
update to log appropriately when one of them is None.

Reply:
done